Resources/Prototypes/Entities/Mobs/Species/skeleton.yml

```diff
@@ -0,0 +1,355 @@
+- type: entity
+  save: false
+  name: Urist McSkelly
+  id: MobSkeletonPersonBase
+  description: A miserable pile of bones.
+  noSpawn: true
+  components:
+  - type: Tag
+    tags:
```

Comment:
are these really so different from humans that they cant inherit from them?

Reply:
The reason they can't inherit from humans is because they don't have hunger or thirst. That's honestly the only reason at this point.

Reply:
This will immediately lead to bugs whenever human gets updated and people don't realise skeletons also need updating so it needs fixing before a merge.

Reply:
I would have the inheritance if it was possible, but there's simply an issue of no possible way of defining components that should not be inherited by the parent. Until such a system exists, realistically the best thing to do is just to stay on top of it and try and update whenever possible. Usually there aren't too many large prs for species so it shouldn't be that bad to manage until a decent solution exists.

Reply:
I'll just cherry-pick from my kepori branch...